ragtime.core/src/ragtime/main.clj

```diff
@@ -35,7 +25,7 @@
   (let [db (core/connection database)]
     (doseq [m (resolve-migrations migrations)]
       (core/remember-migration m))
-    (core/rollback-last db (or (when n (Integer/parseInt n)) 
+    (core/rollback-last db (or (when n (Integer/parseInt n))
```

Comment:
Ah right, I saw the change and thought it had added the "1" at the end. False alarm :)